srcpkgs/xbindkeys/template

```diff
@@ -1,14 +1,19 @@
 # Template file for 'xbindkeys'
 pkgname=xbindkeys
 version=1.8.7
-revision=2
+revision=3
 build_style=gnu-configure
-hostmakedepends="pkg-config"
-makedepends="libX11-devel guile-devel guile gc-devel"
-depends="guile"
+hostmakedepends="automake pkg-config"
+makedepends="libX11-devel guile3-devel"
+depends="guile3"
```

Comment:
Shouldn't that be autodetected?

Reply:
Yes, you are right. Thanks for the correction.